BuildResidentialScheduleFile/tests/build_residential_schedule_file_test.rb

```diff
@@ -42,7 +42,7 @@ def test_stochastic
     assert(!info_msgs.any? { |info_msg| info_msg.include?('RandomSeed') })
     assert(info_msgs.any? { |info_msg| info_msg.include?('GeometryNumOccupants=3.0') })
 
-    sf = SchedulesFile.new(schedules_paths: hpxml.header.schedules_filepaths,
+    sf = SchedulesFile.new(schedules_paths: hpxml.buildings[0].header.schedules_filepaths,
```

Comment:
Sample files now stored schedule filepath at building level, so update tests to look there.